src/java.base/share/classes/java/time/format/Parsed.java

```diff
@@ -132,6 +133,10 @@
      * The parsed zone.
      */
     ZoneId zone;
+    /**
+     * The parsed zone name type.
+     */
+    int zoneNameType = -1;
```

Comment:
Could be an Enum if that helps with readability.

Reply:
Since the index needs some calculation, I left it as `int`. Changed to use the constant fields.